core/common/src/test/java/alluxio/security/LoginUserTestUtils.java

```diff
@@ -29,7 +29,9 @@ private LoginUserTestUtils() {} // prevent instantiation
    * Resets the singleton {@link LoginUser} to null.
    */
   public static void resetLoginUser() {
-    Whitebox.setInternalState(LoginUser.class, "sLoginUser", (String) null);
+    synchronized (LoginUser.class) {
```

Comment:
i don't find this a case. but synchronizing on this to be safe. 
reason is that in `public static void resetLoginUser(String user)` we already `synchronized (LoginUser.class)`. I think we should be consistent.